doc/releases/changelog-dev.md

```diff
@@ -40,11 +40,15 @@
 * The ``qml.QSVT`` template now orders the ``projector`` wires first and the ``UA`` wires second, which is the expected order of the decomposition.
   [(#6212)](https://github.com/PennyLaneAI/pennylane/pull/6212)
 
+* ``qml.lie_closure`` now uses a more stable SVD based linear independence check to avoid running into `LinAlgError: Singular matrix`. Also introduces normalization of basis vectors to avoid exploding coefficients.
```

Comment:
Two somewhat nitpicky comments that might be useful down the road:
- Technically speaking, the change is made to `PauliVSpace`.
  While I agree that `lie_closure` is the functionality that most prominently uses this implementation object, it might be worth commenting that this change is in `PauliVSpace` and hence can also affect other things using it.
- The basis vectors `PauliVSpace.basis` are not being normalized, just the internal `_M` matrix is being normalized, right? Might be good to say "internally introduces a normalization" or something like this, to indicate that we do not actually return normalized objects to the user.